### ISSUE TITLE
Error during evaluation: Check failed: cudnnSetTensorNdDescriptor(handle_.get(), elem_type, nd, dims.data(), strides.data()) == CUDNN_STATUS_SUCCESS (9 vs. 0)batch_descriptor: {count: 64 feature_map_count: 64 spatial: 768 768  value_min: 0.000000 value_max: 0.000000 layout: BatchYXDepth} Fatal Python error: Aborted

### ISSUE THREAD

**Issue by Pari-singh** [google/automl#770]
Hi,

Prior to the recent changes in the repository, I was able to perform the evaluation on my (custom-data) fine-tuned model, however, the class-wise APs reported were 0.0. Now that the author @mingxingtan has changed the code to accomodate the class-wise APs, I am getting the following error running the same script I did before, but with newly cloned repo:

Check failed: cudnnSetTensorNdDescriptor(handle_.get(), elem_type, nd, dims.data(), strides.data()) == CUDNN_STATUS_SUCCESS (9 vs. 0)batch_descriptor: {count: 64 feature_map_count: 64 spatial: 768 768  value_min: 0.000000 value_max: 0.000000 layout: BatchYXDepth}
Fatal Python error: Aborted

Could someone who has faced and resolved this help? thanks

**Comment by mingxingtan** [comment]
How to reproduce this error?

**Comment by Pari-singh** [comment]
This is by running the evaluation on any checkpoints. I was using the following code for eval:

   python main.py --mode=eval 
     --validation_file_pattern=$val 
     --tfrecordfile --model_name=efficientdet-d7x
     --model_dir=$model_dir
     --ckpt=$ckpt 
     --hparams=new_config.yaml 
     --eval_batch_size=1

As you mentioned in the issue #765 we should use train_batch_size even during evaluation, should we keep the train_batch_size the same as what we kept during training or we can change it to 1 or any number which divides our samples perfectly? 

Second question I have is, do we need to give the val_json file if we already have the labels in our tfrecord? If yes, what does the json format expect?
Thanks

**Comment by mingxingtan** [comment]
The eval batch size issue should have fixed. Closing it now.
Feel free to reopen it if you still have issues.
